cilium/cmd/bpf_metrics.go

```diff
@@ -0,0 +1,29 @@
+// Copyright 2017 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+)
+
+// bpfMetricsCmd represents the bpf_metrics command
+var bpfMetricsCmd = &cobra.Command{
+	Use:   "metrics",
+	Short: "L3/L4 metrics",
```

Comment:
If this is the case, then yes, I would recommend rewording the description. @manalibhutiyani perhaps you could post the output of the  `cilium bpf metrics list` and I can help suggest a description?

Reply:
There you go. I will refrain from making cosmetic changes, and if that's ok with you I change this as part of a followup PR.
```
vagrant@runtime1:~/go/src/github.com/cilium/cilium$ sudo cilium bpf metrics     
L3/L4 metrics

Usage:
  cilium bpf metrics [command]

Available Commands:
  list        List L3/L4 metrics

Global Flags:
      --config string   config file (default is $HOME/.cilium.yaml)
  -D, --debug           Enable debug messages
  -H, --host string     URI to server-side API

Use "cilium bpf metrics [command] --help" for more information about a command.
```

```
vagrant@runtime1:~/go/src/github.com/cilium/cilium$ sudo cilium bpf metrics list
KEY                COUNT                 
reason:0 dir:1     count:22 bytes:2102   
reason:131 dir:2   count:22 bytes:1700   
reason:133 dir:2   count:5 bytes:370     
reason:0 dir:2     count:25 bytes:2324   
reason:133 dir:1   count:3 bytes:222     
```

Reply:
Changed to. wording to
`BPF datapath traffic metrics`
thanks!

Reply:
Looking at the output of that CLI, it would be nice to split out into multiple columns for something like:

```
REASON          DIRECTION    PACKETS    BYTES
PolicyDropped   Ingress      1          64
```

...but that can also be followed up separately.

Reply:
Yes, I do plan to do it in the followup PR. But would like the data path bits to be in if everything looks fine.

Reply:
@joestringer Raised #4223 to track this as a bug fix post the feature freeze deadline. Thanks!